pkg/distributors/contracts/interfaces/IMultiDistributor.sol

```diff
@@ -21,17 +21,38 @@ import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/EnumerableSet.so
 import "./IDistributorCallback.sol";
 
 interface IMultiDistributor {
-    // paymentRate and globalTokensPerStake are stored as 18 decimal fixed point values
+    /**
+     * @dev The Distribution struct has its fields packed to minimise the number of storage reads/writes.
+     * Timestamps are stored in 64 bits which ensures that the Sun will explode before they overflow.
+     * `globalTokensPerStake` and `paymentRate` are stored as 18 decimal fixed point numbers,
+     * leaving 128 bits for the integer component allowing us to store values of up to ~10^38
+     *
+     * Slot 1
+     *  - IERC20 stakingToken;
+     * Slot 2
+     *  - IERC20 distributionToken;
+     * Slot 3
+     *  - address owner;
+     *  - uint64 duration;
+     * Slot 4
+     *  - uint256 totalSupply;
+     * Slot 5 (fields always written to together)
+     *  - uint192 paymentRate;
+     *  - uint64 periodFinish;
+     * Slot 6 (fields always written to together)
+     *  - uint192 globalTokensPerStake;
+     *  - uint64 lastUpdateTime;
+     */
```

Comment:
Have you double-checked that this is the actual slot allocation we get?

Reply:
I'm not sure how I would test this directly however I get the expected drop of gas costs in the benchmarks that I would expect from this allocation.

Based on this along with the specification as described [here](https://docs.soliditylang.org/en/v0.8.10/internals/layout_in_storage.html) I don't see any reason to doubt that we're getting this configuration but if you know an easy way to check directly then happy to check.